src/main/resources/assets/design-system/icons.scss

```diff
@@ -125,3 +125,11 @@
 .sci-icon-magnifier-text-soft {
   background-image: url(svgResource('/assets/images/icons/magnifier.svg', #212529));
 }
+
+.sci-icon-trash {
+  background-image: url(svgResource('/assets/images/icons/trash.svg', #53595D));
```

Comment:
Shouldn't this by default use the same colour as the other icons? Can you use for example opacity to make it lighter in OXOMI?

Reply:
Yes, this also doesn't seem to be a "sirius color".
Would prefer to use a color suffix appended to the class like the one above.
I think using text-soft should also be fine for the icons in the watchlist, but feel free to post a screenshot after the change. Its no problem to add additional color variants of the icons with different color suffixes.